config/aprsc.conf

```diff
@@ -27,5 +27,3 @@ HTTPStatusOptions ShowEmail=1
 # After configuring the rest of the settings, remove this bad command
 # from the configuration file. It's here only to avoid starting the
 # server up accidentally with an invalid configuration.
-MagicBadness	42.7
```

Comment:
I think we should leave the "MagicBadness" since the user need to set a correct "ServerId" and "PassCode", the 
 "MagicBadness" make sures the user at least does something in the file.